.circleci/config.yml

```diff
@@ -125,14 +129,6 @@ jobs:
       - update_node:
           os: linux
 
-      # We want our unit tests to run with the lts version of node.
-      - node/install:
-          install-npm: false
-          lts: true
-
-      # Log the version of node we're using after the upgrade.
-      - run: node -v
-
```

Comment:
Apparently this snippet slipped through the cracks when I cleaned up the file earlier. It needs to go.